app/workers/hg/message_worker.rb

```diff
@@ -26,7 +26,12 @@ def perform(user_id, redis_namespace, bot_class_name)
       until raw_message.empty?
         # Instantiate a message object with the raw message from the queue.
         message = Facebook::Messenger::Incoming::Message.new(raw_message)
-
+        
+        # Instantiate chatbase client
+        if ENV['CHATBASE_API_KEY']
+          @client = ChatbaseAPIClient.new
```

Comment:
Not a big deal, but usually you'll see one-line these written with a "modifer `if`" (also works with `unless`):

```ruby
@client = ChatbaseAPIClient.new if ENV[`CHATBASE_API_KEY`]
```

Install the [CodeClimate Browser Extension](3963243369aad4685f0f3b623f902887beefbb6d) if you want to see these recommendations inline in your PR diff:

![](http://share.voxable.io/051A0R2w0r0a/Screen%20Shot%202018-01-27%20at%202.02.47%20PM.png)